integrations/launchdarkly/client.py

```diff
@@ -135,18 +135,48 @@ async def fetch_environments_for_project(
             environments.extend(updated_batch)
         return environments
 
+    async def get_feature_flag_status(
+        self, projectKey: str, featureFlagKey: str
+    ) -> dict[str, Any]:
+        endpoint = f"flag-status/{projectKey}/{featureFlagKey}"
+        feature_flag_status = await self.send_api_request(endpoint)
+        return feature_flag_status
+
+    async def append_feature_flag_statuses(
+        self, feature_flags: list[dict[str, Any]]
+    ) -> list[dict[str, Any]]:
+        tasks = [
+            self.get_feature_flag_status(
+                feature_flag["__projectKey"], feature_flag["key"]
+            )
+            for feature_flag in feature_flags
+        ]
+        statuses = await asyncio.gather(*tasks)
+        for feature_flag, status in zip(feature_flags, statuses):
+            feature_flag["__status"] = status
+        return feature_flags

```

Comment:
Wont it send tons of requests?
id prefer adding the environment and feature flag status kinds and export the seperately